public/video-ui/src/components/Video/Display.js

```diff
@@ -119,11 +120,26 @@ class VideoDisplay extends React.Component {
     const errors = this.props.checkedFormFields[formName] ? this.props.checkedFormFields[formName] : {};
     return Object.keys(errors).some(field => {
       const value = errors[field];
-      return value.length !== 0;
+      return value !== null;
     });
 
   };
 
+  updateFormErrors = (notification) => {
+    if (notification.type === 'error') {
+      this.props.formErrorActions.updateFormErrors(notification);
+    } else {
+      this.props.formErrorActions.updateFormErrors(null);
+    }
+  }
+
+  validateDescription = (description) => {
+    if (!description) {
+      return new FieldNotification('required', 'It is recommeded you fill in this field for seo', 'warning');
```

Comment:
minor: could have [static getters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static) on the `FieldNotification` class for `'required'` and `'warning'`, making this line (and others below) read something like:

```js
return new FieldNotification(FieldNotification.required, 'It is recommeded you fill in this field for seo', FieldNotification.warning);
```